README.md

```diff
@@ -283,6 +293,17 @@ Note: **Only** on macOS versions **before** 10.15.7 you might need to add this e
 #### "Port forwarding does not work"
 Privileged ports (1-1023) cannot be forwarded. e.g., you have to use 8080, not 80.
 
+#### stuck on "Waiting for the essential requirement 1 of X: "ssh"
+
+libslirp v4.6.0 used by QEMU is known to be [broken](https://gitlab.freedesktop.org/slirp/libslirp/-/issues/48).
+If you have libslirp v4.6.0 in `/usr/local/Cellar/libslirp`, you have to downgrade it to v4.5.0:
+
+```console
+brew uninstall --ignore-dependencies libslirp
+wget https://raw.githubusercontent.com/Homebrew/homebrew-core/f2a53ffe2362d2a7e55fd1a8a9bc71c1ec15bb00/Formula/libslirp.rb
+brew install ./libslirp.rb
+```
```

Comment:
Don't you need a `brew pin libslirp` too, to avoid auto-upgrading it the next time you run `brew upgrade`?

Reply:
Yes, thanks, opened https://github.com/AkihiroSuda/lima/pull/61